com.unity.render-pipelines.high-definition/Documentation~/HDR-Output.md

```diff
@@ -0,0 +1,96 @@
+| **Neutral HDR Range Reduction Mode** | The tonemapping curve used for the Neutral tonemapper. The options are:<br />- BT2390: Uses a curve defined by the BT2390 broadcasting recommendations. (Default)<br />- Reinhard: A very simple tonemapping operator.<br /><br />This option is available only when Additional Properties are displayed. |
+| **Hue Shift Amount**                 | Determines how much hue preservation is desired. When the value is 0, the tonemapper will try to preserve the hue of the content as much as possible, tonemapping only the luminance. However, some content might have been authored assuming that hue-shift will happen when all color channels are tonemapped independently. To recover the hue-shift behaviour this slider can be moved toward See note below for more information. |
+| **Detect Paper White**               | Whether the paper white value is detected from the data communicated by the display device. We strongly suggest to provide a calibration screen to let the user set this value depending on their screen and viewing experience. |
+| **- Paper White**                    | Paper white value (in nits) set when it is not automatically detected from the display. |
+| **Detect Brightness Limits**         | Whether the minimum and maximum brightness values are detected from the data communicated by the display device. While using these values as detected lead to more precise results than detecting paperwhite values, we still suggest to use a calibration screen. |
+| **- Min Nits**                       | Minimum brightness supported by the device.                  |
+| **- Max Nits**                       | Maximum brightness supported by the device.                  |
+
+While Hue-preserving tonemapping (i.e. Hue Shift Amount set to 0) will better preserve the content colors, sometimes content is authored to rely on the hue shifts that high brightness will produce. A typical example can be for example a very bright flame VFX. The image below shows on the left a flame with Hue Shift Amount set to 0 and on the right the same flame with the hue shift preserved (Hue Shift Amount set to 1).
+
+ <img src="C:\Users\franc\Documents\Github\SRP\com.unity.render-pipelines.high-definition\Documentation~\Images\HDR-Output-HueShift.png" alt="HDR-Output-HueShift" style="zoom:67%;" />
+*Image modified to show the issue clearly.*
+
+The right choice depends on your content, but we suggest to make sure your content is authored to work well without relying on Hue-shift.
+
+#### ACES
+
+Contrary to the neutral version, the ACES tonemapping option has fixed presets. These are meant to target screens with 1000, 2000 and 4000 nits maximum brightness. We suggest to use calibration screen to detect the right preset for the user consuming the content.
+
+![HDR-Output-ACES](C:\Users\franc\Documents\Github\SRP\com.unity.render-pipelines.high-definition\Documentation~\Images\HDR-Output-ACES.png)
```

Comment:
The table means that we don't necessarily need the screenshot. 
https://unity-docs.gitbook.io/style-guide/format/images/screenshots